package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "typescript-urban-airship-client",
-  "version": "0.3.2",
+  "version": "0.3.3",
```

Comment:
Guess I only updated the package.json and forgot the lock file 😄 , however, we'll want to update both of these to 0.3.4 for the next release containing these changes.